.rspec

```diff
@@ -1 +1,3 @@
--fd --color
+--format documentation
+--color
+--order defined
```

Comment:
Unless you have a specific reason, you should let your specs run in random order. This can find edge cases sometimes.